### ISSUE TITLE
Erreur dans le mappage des leds de la deuxième partie du cube

### ISSUE THREAD

**Issue by lewisjaggi** [LukasCH09/cubeSimulator#1]
Dans la deuxième partie du cube il y a un chevauchement des adresses des leds ce qui provoque une erreur dans le simulateur.
```json
{"axis_x": 25 , "axis_y": 20 , "universe": 22 ,"channel": 1 },
{"axis_x": 25 , "axis_y": 22 , "universe": 22 ,"channel": 58 },
``` 
Coordonnée: (25,20) démarre au début de l'univers 22, adresses de la première led (1,2,3).
Il faut 27*3=81 adresses par coordonnée (x,y) (ledStrip du cube) donc les adresses de la dernière led sont (79,80,81).
On voit qu'aux coordonnées (25,22),  on est dans l'univers 22 et que la première led est à l'adresse 58. Il y a donc un chevauchement.

**Comment by lewisjaggi** [comment]
Ce n'est pas qu'une simple erreur de frappe, si on change le 52 par 82 l'erreur ce répercute plus bas.
Il faut faire plus d'investigations.

**Comment by LukasCH09** [comment]
Effectivement. Le problème vient du fichier json du code python où le problème est déjà présent.
@mylag je n'ai plus la licence pour madmapper, si tu peux y jeter un coup d'oeil?
